lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -291,6 +291,18 @@
     }
 }
 
+.data-disclaimer-modal {
```

Comment:
This part does not belong to this pr right? this is the artifact we get from CAM-12 that will eventually go away after we merge CAM-12 and rebase

Reply:
Yes, This is CAM-12